### PR TITLE
Py2DeprecateNotice

### PR DESCRIPTION
**Pull request by markgalpin**

### CODE REVIEW

**Comment by markgalpin** [comment]
For some reason when I run this on the test jekyll server the post isn't showing up?  What did I forget?

**Comment by lasote** [comment]
You forgot to travel to the future.

File: _posts/2018-08-13-Its-Time-To-Deprecate-Python-2.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+comments: false
+title: "It's Time to Deprecate Python 2"
+---
+
+*"When in the Course of human events, it becomes necessary for one people to dissolve the ... bands which have connected them with another ... a decent respect to the opinions of mankind requires that they should declare the causes which impel them to the separation."*
+
+US Declaration of Independence, Thomas Jefferson
```
Comment (lasote) [inline _posts/2018-08-13-Its-Time-To-Deprecate-Python-2.markdown]: I would hardly recommend removing non-technical aspects like the Thomas Jefferson quote.

Comment (memsharded) [reply _posts/2018-08-13-Its-Time-To-Deprecate-Python-2.markdown]: Yes, agree.

Comment (markgalpin) [reply _posts/2018-08-13-Its-Time-To-Deprecate-Python-2.markdown]: Fair enough.  I think a little humor doesn't hurt -- its a blog, not documentation -- but I suppose it is probably a fairly culturally specific joke.

**Pull request merged**